recipes/libraw/all/test_package/CMakeLists.txt

```diff
@@ -3,6 +3,11 @@ project(test_package LANGUAGES CXX)
 
 find_package(libraw REQUIRED CONFIG)
 
+# libraw/0.19.5 uses std::auto_ptr 
+if (libraw_VERSION VERSION_EQUAL "0.19.5")
+    set(CMAKE_CXX_STANDARD 11)
+endif()
```

Comment:
auto_ptr has been removed in C++17, so C++14 is valid (please fix your check_max_cppstd)

Reply:
But honestly, you should remove this version.